src/ast/query.rs

```diff
@@ -276,6 +280,8 @@ impl fmt::Display for Table {
 #[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
 #[cfg_attr(feature = "visitor", derive(Visit, VisitMut))]
 pub struct Select {
+    /// SELECT
+    pub select_token: IgnoreField<TokenWithLocation>,
```

Comment:
For the comment we can probably be more descriptive to mention that this points to the position of the SELECT keyword? (similar for others probably if we want to make it more obvious which part of the statement is being referred to)

Reply:
yup 👍